### ISSUE TITLE
Link not found error

### ISSUE THREAD

**Issue by ADITYADAS1999** [openshift/okd-machine-os#454]
At README section the building process links are not working.

![5555](https://user-images.githubusercontent.com/58718316/193439581-22138036-387f-4817-a702-058aa31cb2d2.PNG)

**Comment by vrutkovs** [comment]
Right, this is a build procedure for release-4.11 branch. We'll rework the README after layering rework stabilizies
